components/FeatureHighlight/examples/FeatureHighlightShownViewExample.m

```diff
@@ -58,8 +58,8 @@ - (void)didTapButton:(id)sender {
   [MDCFeatureHighlightAccessibilityMutator mutate:vc];
```

Comment:
Can you remove the imports/podspec/BUILD deps?

Reply:
Oh whoops I just saw this :/ Will do in follow up.